process_witness.py

```diff
@@ -233,6 +233,10 @@ def processWitness(witness, benchmark, bitwidth):
       a = re.sub(r'\\result', '__SV_COMP_result', a)
       # we may be missing typedefs used in type casts
       a = re.sub(r'\([a-zA-Z0-9_]+\s*\*\)', '(int*)', a)
+      # terminate the expression with semicolon (if needed),
+      # so that the parser can correctly parse the code
+      if a[-1] != ';':
+          a += ';'
```

Comment:
I believe we first need to trim the string to strip off space or just unconditionally always append the ';'.

Nit pick: this code uses 2-space indent (for historic reasons..).

Reply:
Yes, you're right. The later (unconditionally appending semicolon) seems nicer to me (actually, that way I had it written first, but then I changed it). So can I go with it? Or you want to strip the string?

Reply:
Yes, go with the unconditional append - I think that should be fine for all the remaining code.